bazel/external_crates.bzl

```diff
@@ -648,7 +651,9 @@ def external_crates_repository(name, cargo_lockfile, lockfile, sanitizers_enable
                 default_features = False,
             ),
             "ic-wasm": crate.spec(
-                version = "^0.8.4",
+                git = "https://github.com/dfinity/ic-wasm", # DO NOT MERGE
+                rev = "e6b26402f4aa053903ffe4cb3ba2dac5f9dca89b", # DO NOT MERGE
+                # DO NOT MERGE version = "^0.8.4",
```

Comment:
Is this PR meant to be a draft or should IDX review this? If the latter, why the `DO NOT MERGE` comments?

Reply:
Ok I see you converted the PR to draft.

Reply:
This is draft.

I took it out of draft briefly in the hopes that it would cause the tests to actually run.

I tried to remove reviewers, but it won't let me.